bucket/python.json

```diff
@@ -1,8 +1,9 @@
 {
     "homepage": "https://www.python.org/",
+    "description": "A programming language that lets you work quickly and integrate systems more effectively.",
     "license": "Python-2.0",
     "version": "3.7.3",
-    "depends": "dark",
+    "depends": "main/dark",
```

Comment:
No need to prefix it with main. Everything will behave completely same as before

Reply:
Okay, I thought every 'main' manifest should be explicit writen.